.editorconfig

```diff
@@ -9,3 +9,6 @@ indent_size = 2
 indent_style = space
 insert_final_newline = true
 trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
allows line break in markdown documents (see `  ` double spaces in changelog entry from this PR 👇 )